components/card/test/card.visual-diff.js

```diff
@@ -28,22 +27,22 @@ describe('d2l-card', () => {
 		{ name: 'align-center', selector: '#align-center' },
 		{ name: 'badge', selector: '#badge' },
 		{ name: 'actions', selector: '#actions' },
-		{ name: 'actions-focus', selector: '#actions', action: (selector) => page.$eval(`${selector} > d2l-button-icon`, (elem) => forceFocusVisible(elem)) },
+		{ name: 'actions-focus', selector: '#actions', action: (selector) => focusWithKeyboard(page, `${selector} > d2l-button-icon`) },
 		{ name: 'actions-rtl', selector: '#actions-rtl' },
-		{ name: 'no-link-focus', selector: '#header-content', action: (selector) => page.$eval(selector, (elem) => elem.focus()) },
+		{ name: 'no-link-focus', selector: '#header-content', action: (selector) => focusWithMouse(page, selector) },
```

Comment:
I'm surprised that this test was working as-is (and doesn't prompt a changed visual-diff), but looking closer it makes sense. It's entirely coincidental - `visualDiff.resetFocus()` was entering mouse mode, and `forceFocusVisible` wasn't entering keyboard mode, so it just worked out that this test would have been in mouse mode.

It's nice to be making these much more explicit!